decidim-core/app/controllers/decidim/errors_controller.rb

```diff
@@ -10,6 +10,16 @@ def not_found
     end
 
     def internal_server_error
+      @method = request.request_method
+      @reference_id = Decidim::LogReferenceGenerator.new(request).generate_reference
+      info_hash = {
```

Comment:
I think you could just make the `info_hash` an instance variable to pass it to the view.

Then, you could create the table using the `@info_hash` variable, so that you wouldn't have to repeat the same code in the view.